### PR TITLE
Added support for a local Vagrantfile.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**

### CODE REVIEW

**Comment by Decipher** [comment]
@alexdesignworks Change will also need to be made to https://github.com/beetboxvm/beetbox/blob/master/provisioning/packer/includes/Vagrantfile

File: provisioning/packer/includes/Vagrantfile [hunk 1]
```diff
@@ -174,3 +174,7 @@ if cwd != nil && Digest::MD5.file("#{cwd}/#{vagrantfile}").hexdigest == "ef550c4
     end
   end
 end
+
+# Load local Vagrantfile, if exists.
+include_vagrantfile = File.expand_path('../Vagrantfile.local', __FILE__)
```
Comment (thom8) [inline provisioning/packer/includes/Vagrantfile]: @Decipher this won't work in the pre-packaged `Vagrantfile` as `File.expand_path` will expand to the box path.

Comment (Decipher) [inline provisioning/packer/includes/Vagrantfile]: There are variables already in this file that will give you the correct path, it just needs to be changed to the appropriate var, which is `cwd`.

Comment (thom8) [inline provisioning/packer/includes/Vagrantfile]: `cwd` exists in both so this line could be update to `include_vagrantfile = "#{cwd}/Vagrantfile.local"`

**Pull request merged**